tests/periph_timer/main.c

```diff
@@ -35,10 +37,18 @@
 #define CHAN_OFFSET         (5000U)     /* fire every 5ms */
 #define COOKIE              (100U)      /* for checking if arg is passed */
 
-static volatile int fired;
-static volatile uint32_t sw_count;
-static volatile uint32_t timeouts[MAX_CHANNELS];
-static volatile unsigned args[MAX_CHANNELS];
+#ifndef XTIMER_HZ
+#define XTIMER_HZ MHZ(1)
```

Comment:
Just a minor style improvement.

```suggestion
#define XTIMER_HZ         MHZ(1)
```

I can try on usb-kw41z, remote, hifive1b.